test/manual/algo/lin_reg.py

```diff
@@ -7,8 +7,8 @@
 import zipfile
 from sklearn import metrics
 
-DATA_DIR = "datasets"
-RESULTS_DIR = "results"
+DATA_DIR = "/datasets"
+RESULTS_DIR = "/results"
```

Comment:
this absolute path will work on docker but not on python or python or bin. on docker can you have the algo mount be in a relative path to datasets and results mounts

Reply:
Done